### ISSUE TITLE
How about a live filter?

### ISSUE THREAD

**Issue by universalhandle** [twomice/org.civicrm.extensionsui#4]
The "Add new" list gets long fast, and as we get more extensions whitelisted, it's only going to get longer. What about a live filter on names, descriptions, compatibility? We could even trivially set a sane default on the filter of the current CiviCRM version so that the initial view is of only extensions compatible with your version. There's no reason the live filters would need to be limited to the "Add new" tab; we could use them across all tabs.

**Comment by mattwire** [comment]
I was thinking the same :+1:

**Comment by twomice** [comment]
Yep, +1 me too. That's for a Phase II, though. Phase I is just to get feature parity with the existing interface and make easy wins. Someone did create some mockups a few months back that contained filters and other cool things. Anyone remember where those are?
